pkg/applicationserver/config.go

```diff
@@ -54,9 +45,9 @@ type InteropConfig struct {
 
 // Config represents the ApplicationServer configuration.
 type Config struct {
-	LinkMode       string                    `name:"link-mode" description:"Mode to link applications to their Network Server (all, explicit)"`
 	Devices        DeviceRegistry            `name:"-"`
 	Links          LinkRegistry              `name:"-"`
+	Distribution   DistributionConfig        `name:"distribution" description:"Distribution configuration"`
```

Comment:
Please document configuration changes

Reply:
Also, we have to add a dummy flag `--as.link-mode` to avoid breaking existing deployments that do use that flag.